src/main/java/graphics/opengl/OpenGLUtils.java

```diff
@@ -256,4 +274,25 @@ public static void drawTextRed(String text, float x, float y) {
 		//texShader.use();
 	}
 
+	public static void checkGLError() {
+		int glError = glGetError();
+		String errorName = null;
+		if(glError != 0) {
+			if(glError == GL_INVALID_ENUM) errorName = "GL_INVALID_ENUM";
+			if(glError == GL_INVALID_VALUE) errorName = "GL_INVALID_VALUE";
+			if(glError == GL_INVALID_OPERATION) errorName = "GL_INVALID_OPERATION";
+			if(glError == GL_STACK_OVERFLOW) errorName = "GL_STACK_OVERFLOW";
+			if(glError == GL_STACK_UNDERFLOW) errorName = "GL_STACK_UNDERFLOW";
+			if(glError == GL_OUT_OF_MEMORY) errorName = "GL_OUT_OF_MEMORY";
+			if(glError == GL_INVALID_FRAMEBUFFER_OPERATION) errorName = "GL_INVALID_FRAMEBUFFER_OPERATION";
+			if(glError == GL_CONTEXT_LOST) errorName = "GL_CONTEXT_LOST";
+			if(glError == GL_TABLE_TOO_LARGE) errorName = "GL_TABLE_TOO_LARGE";
```

Comment:
Maybe make a switch here

Reply:
sure but performance doesn't matter here, make it so it looks nice

Reply:
switches look nice enough